src/lib/pages/backoffice/Document/DocumentForms/DocumentEditor/DocumentForm/components/LicensesField.js

```diff
@@ -2,18 +2,70 @@ import React from 'react';
 import { AccordionField } from '@forms/core/AccordionField';
 import { VocabularyField } from '@forms/core/VocabularyField';
 import { invenioConfig } from '@config';
+import { ArrayField } from '@forms/core/ArrayField';
+import { DeleteActionButton } from '@forms/components';
+import { GroupField } from '@forms/core/GroupField';
+import { StringField } from '@forms/core/StringField';
+import { TextField } from '@forms/core/TextField';
+import _pickBy from 'lodash/pickBy';
 
 export class LicensesField extends React.Component {
+  serializer = hit => {
+    const {
+      metadata: {
+        id,
+        text,
+        data: { status, maintainer, url },
+      },
+    } = hit;
+    const value = { id, status, maintainer, title: text, url };
+    return {
+      key: id,
+      value: _pickBy(value, o => o), // Remove empty fields
```

Comment:
Not sure if this is anti-pattern or not. But it was the only solution I found to store an object as the value of a selection input (as given by the requirement).